pom.xml

```diff
@@ -1,293 +1,373 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns="http://maven.apache.org/POM/4.0.0"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.eclipse.ee4j</groupId>
-	<artifactId>cargo-tracker</artifactId>
-	<version>2.2-SNAPSHOT</version>
-	<packaging>war</packaging>
-
-	<name>Eclipse Cargo Tracker</name>
-	<description>
+    <groupId>org.eclipse.ee4j</groupId>
+    <artifactId>cargo-tracker</artifactId>
+    <version>2.1-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>Eclipse Cargo Tracker</name>
+    <description>
         The project demonstrates how you can develop applications with the Jakarta EE platform using
         widely adopted architectural best practices like Domain-Driven Design (DDD).
     </description>
-	<url>https://eclipse-ee4j.github.io/cargotracker/</url>
+    <url>https://eclipse-ee4j.github.io/cargotracker/</url>
+
+    <organization>
+        <name>Eclipse Foundation</name>
+        <url>https://www.eclipse.org/org/foundation/</url>
+    </organization>
 
-	<organization>
-		<name>Eclipse Foundation</name>
-		<url>https://www.eclipse.org/org/foundation/</url>
-	</organization>
+    <issueManagement>
+        <system>github</system>
+        <url>https://github.com/eclipse-ee4j/cargotracker/issues</url>
+    </issueManagement>
 
-	<issueManagement>
-		<system>github</system>
-		<url>https://github.com/eclipse-ee4j/cargotracker/issues</url>
-	</issueManagement>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.report.sourceEncoding>UTF-8</project.report.sourceEncoding>
 
-	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<project.report.sourceEncoding>UTF-8</project.report.sourceEncoding>
+        <maven.compiler.release>17</maven.compiler.release>
```

Comment:
We need to stick to Java SE 11 here. That's what Jakarta EE 10 actually requires.

Reply:
Change to Java 11